### PR TITLE
Make it easier to do cross-platform builds in pipelines

### PR DESCRIPTION
**Pull request by jameshalsall**

### CODE REVIEW

File: harness/config/commands.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ command('networks external'):
 command('build'): |
   #!bash(workspace:/)|@
   ws external-images pull
-  passthru docker-compose build app
+  passthru docker buildx build -f ./.my127ws/docker/image/app/Dockerfile .
```
Comment (andytson-inviqa) [inline harness/config/commands.yml:36]:
```suggestion
  passthru docker buildx bake --load
```
only while single platform image though

File: harness/attributes/common.yml [hunk 2]
```diff
@@ -48,6 +48,9 @@ attributes.default:
     external_image_registries: []
     repository: = @("workspace.name")
     tagPrefix: = ''
+    platforms:
+      production:
+      - "linux/amd64"
```
Comment (jameshalsall) [inline harness/attributes/common.yml:53]: need to document these in https://github.com/inviqa/harness-go/blob/master/docs/harness-attributes.md

File: harness/config/confd.yml [hunk 3]
```diff
@@ -3,14 +3,14 @@ confd('harness:/'):
...
   - { src: docker/image/app/Dockerfile }
-  - { src: docker/image/app/Dockerfile.prod }
   - { src: docker/image/app/root/lib/task/gocyclo.sh }
   - { src: docker/image/app/root/lib/task/gosec.sh }
   - { src: docker/image/app/root/lib/task/fmt/check.sh }
   - { src: docker/image/app/root/lib/task/modules/after.sh }
   - { src: docker/image/app/root/lib/task/modules/before.sh }
   - { src: docker/image/app/root/lib/task/modules/check.sh }
   - { src: docker-compose.yml, dst: workspace:/docker-compose.yml }
+  - { src: _twig/docker-bake.hcl, dst: workspace:/docker-bake.hcl }
```
Comment (andytson-inviqa) [inline harness/config/confd.yml:13]: I'd for convention have docker-bake.hcl in the top of the harness

**Review by andytson-inviqa — COMMENTED** [review]
can we do the scratch fix the same way as I did in harness-base-php?

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**